src/main/java/com/github/appreciated/apexcharts/ApexCharts.java

```diff
@@ -22,8 +22,20 @@
 @JsModule("./com/github/appreciated/apexcharts/apexcharts-wrapper.js")
 @CssImport(value = "./com/github/appreciated/apexcharts/apexcharts-wrapper-styles.css", id = "apex-charts-style")
 public class ApexCharts extends PolymerTemplate<ApexChartsModel> implements HasSize, HasStyle, HasTheme {
+    private final ObjectMapper objectMapper;
 
     public ApexCharts() {
+        this(null);
+    }
+
+    public ApexCharts(ObjectMapper objectMapper) {
```

Comment:
Thanks @F43nd1r for the PR. While I don't understand how customizing the ObjectMapper helps, also I must admit I'm also not too satisfied with the logic the ObjectMapper is being set. 
I think a default constructor with the default ObjectMapper, and a seperate setter Method to overwrite ObjectMapper would be the better option. I hope you can relate, sorry for being so picky.

Reply:
Primary use case for me is registering modules to handle java.time types.
```
   objectMapper.registerModule(new JavaTimeModule());
```

Reply:
Thank you very much 👍

Reply:
Thanks for the speedy response time! And don't worry about requesting style changes, can absolutely relate to keeping everything consistent.